daemon/cmd/daemon_main.go

```diff
@@ -1566,7 +1567,12 @@ func registerDaemonHooks(lc fx.Lifecycle, shutdowner fx.Shutdowner, cfg DaemonCe
 		clientset.Disable()
 	}
 
-	lc.Append(fx.Hook{
+	// Set the global LocalNodeStore. This is to retain the API of getters and setters
+	// defined in pkg/node/address.go until uses of them have been converted to use
+	// LocalNodeStore directly.
+	node.SetLocalNodeStore(localNode)
```

Comment:
I don't understand this diff. It seems separate from the refactoring of the hive / cells, no?

Reply:
It made sense to move it here from `root.go`. Why this ended up in this PR was because I had implicitly relied on ordering of the invokes and when I accidently moved the `registerDaemonHooks` to be before the `SetLocalNodeStore` invoke in `root.go` things broke. I'll break it into its own commit.